packages/contractkit/src/wrappers/Attestations.ts

```diff
@@ -215,37 +219,63 @@ export class AttestationsWrapper extends BaseWrapper<Attestations> {
 
     const result = await this.contract.methods.getCompletableAttestations(phoneHash, account).call()
 
+    const results = await concurrentMap(
+      5,
+      parseGetCompletableAttestations(result),
+      this.isIssuerRunningAttestationService
+    )
+
+    return results.map((_) => (_.isValid ? _.result : null)).filter(notEmpty)
+  }
+
+  /**
+   * Returns an array of issuer addresses that were found to not run the attestation service
+   * @param phoneNumber
+   * @param account
+   */
+  async getNonCompliantIssuers(phoneNumber: string, account: Address): Promise<Address[]> {
```

Comment:
Seems like this is exactly the same as the getActionableAttestations method but with the ternary in the last line reversed. Can we DRY this up?